src/api/app/views/webui/package/_live_build_log_controls.html.haml

```diff
@@ -30,3 +30,7 @@
               class: 'link_abort_build live-link-action btn-danger d-none') do
       %i.far.fa-times-circle
       Abort Build
+
+  = link_to('#', class: 'link_notifications live-link-action btn-secondary ml-auto', title: 'Request permission for browser notifications') do
+    %i.far.fa-bell
+    Browser Notifications
```

Comment:
I would rephrase this as an action, like 'Request Browser Notifications', then once disabled I would add a help title like 'Browser Notifications Granted' or something like this so the user knows why it's greyed out

Reply:
Okay, but I would change the title to something else so it's clear that you cannot click on that button because you already did it.

Reply:
I renamed the button to 'Request Browser Notifications'. Please have a look.

Reply:
> Okay, but I would change the title to something else so it's clear that you cannot click on that button because you already did it.

Now it will show either "Browser Notifications Granted" or "Browser Notifications Denied". I will extract the duplicated code into a function later. Please have a look.